app/scripts/modules/appengine/helpContents/appengineHelpContents.ts

```diff
@@ -32,6 +32,14 @@ module(APPENGINE_HELP_CONTENTS_REGISTRY, [
                 (Spinnaker load balancer) will be stopped. This option will be respected only if this server group will
                 be receiving all traffic and the previous server group is using manual scaling.`,
       },
+      {
+        key: 'appengine.loadBalancer.shardBy.cookie',
```

Comment:
Is shard the official term here? Or does it go further and really implement sticky sessions?